package/update.sh

```
@@ -1,3 +1,3 @@
 #!/bin/bash
-cd /srv/www/website/database
+cd /srv/www/abscissa/database
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)